hyosin-Jang/230509/테이블해시함수.js

```diff
@@ -0,0 +1,15 @@
+function solution(data, col, row_begin, row_end) {
+  data.sort((a, b) => {
+    if (a[col - 1] === b[col - 1]) {
+      return b[0] - a[0];
+    }
+    return a[col - 1] - b[col - 1];
+  });
```

Comment:
```suggestion
    data.sort((a, b)=>{
        return a[col-1]===b[col-1]? b[0]-a[0] : a[col-1]-b[col-1];
    })
```
취향차이긴 하지만, 각 조건문에서 수행하는 코드가 한 문장밖에 없다면, 삼항연산자를 이용하면 간결하게 코드를 짤 수 있을 것 같아요!